### ISSUE TITLE
Missing $ symbol in the activity part of the appfilter.xml file

### ISSUE THREAD

**Issue by Primokorn** [zixpo/candybar-sample#38]
Hello,

Here is an example from the appfilter.xml of an icon request:
`<item component="ComponentInfo{app.rvx.android.youtube/com.google.android.youtube.app.honeycomb.ShellHomeActivity}" drawable="revanced_extended" />`

The right activity is: `com.google.android.youtube.app.honeycomb.Shell$HomeActivity`. This is the one we have with the built-in bug report tool so it seems there's a problem with the generation of the appfilter.xml

CandyBar Version : 3.16.2

Thank you.

**Comment by Primokorn** [comment]
A list of all apps for which I noticed a $ symbol in the activity:

````
com.discord/com.discord/com.discord.screens.ScreenAux$Main
com.dualboot.apps.beach/com.dualboot.apps.beach/com.dualboot.apps.beach.Main$Activity
com.dualboot.apps.forest/com.dualboot.apps.forest/com.dualboot.apps.forest.Main$Activity
com.dualboot.apps.ocean/com.dualboot.apps.ocean/com.dualboot.apps.ocean.Main$Activity
com.dualboot.apps.space/com.dualboot.apps.space/com.dualboot.apps.space.Main$Activity
com.etisooy.carpark/com.etisooy.carpark/com.etisooy.carpark.wdgen.GWDPCarPark$WDLanceur
com.ezstudio68.comicvnnet/com.ezstudio68.comicvnnet/com.ezstudio68.comicvnnet.MainActivity$SplashActivity
com.gojuno.rider/com.gojuno.rider/com.gojuno.rider.presentation.Main$Activity
com.google.android.ogyoutube/com.google.android.ogyoutube/com.google.android.ogyoutube.app.honeycomb.Shell$HomeActivity
com.google.android.youtube/com.google.android.youtube/com.google.android.youtube.app.honeycomb.Shell$HomeActivity
com.hb.dialer.free/com.hb.dialer.free/com.hb.dialer.ui.ShcStartActivity$Contacts
com.hb.dialer.free/com.hb.dialer.free/com.hb.dialer.ui.ShortcutStartActivity$Contacts
com.hb.dialer.free/com.hb.dialer.free/com.hb.dialer.ui.ShortcutStartActivity$Favorites
com.huawei.sos/com.huawei.sos/com.huawei.sos.Settings$EmergencySettingsActivity
com.karinzebox.fiches_psc_1_inzebox/com.karinzebox.fiches_psc_1_inzebox/com.karinzebox.fiches_psc_1_inzebox.wdgen.GWDPFiches_PSC_1_InZeBox$WDLanceur
com.lara.android.youtube/com.lara.android.youtube/com.google.android.youtube.app.honeycomb.Shell$HomeActivity
com.LogOvoide.Horoscope_du_couple/com.LogOvoide.Horoscope_du_couple/com.LogOvoide.Horoscope_du_couple.wdgen.GWDPP06_T$WDLanceur
com.lonelycatgames.PM/com.lonelycatgames.PM/com.lonelycatgames.PM.Fragment.AccountListFragment$AccountListActivity
com.magicv.airbrush/com.magicv.airbrush/com.magicv.airbrush.common.StartupActivity$VIP
com.masociete.misterweb/com.masociete.misterweb/com.masociete.misterweb.wdgen.GWDPMisterWeb$WDLanceur
com.mycompany.deped_commons/com.mycompany.deped_commons/com.mycompany.deped_commons.wdgen.GWDPDepEd_Commons$WDLanceur
com.pgsolutions.smartcity/com.pgsolutions.smartcity/com.pgsolutions.smartcity.wdgen.GWDPSmartCity$WDLanceur
com.poulpy.vwtrip/com.poulpy.vwtrip/com.poulpy.vwtrip.wdgen.GWDPSave_my_VW_trip$WDLanceur
com.pvanced.android.youtube/com.pvanced.android.youtube/com.google.android.youtube.app.honeycomb.Shell$HomeActivity
com.threebeesonline.panacloc/com.threebeesonline.panacloc/com.threebeesonline.panacloc.wdgen.GWDPJeffPanacloc$WDLanceur
fr.aexae.icomete/fr.aexae.icomete/fr.aexae.icomete.wdgen.GWDPiComete$WDLanceur
fr.ceicom.smartccpb/fr.ceicom.smartccpb/fr.ceicom.smartccpb.wdgen.GWDPSMARTCCPB$WDLanceur
info.intrasoft.habitgoaltracker/info.intrasoft.habitgoaltracker/info.intrasoft.goalachiver.list.GoalList$ListActivity
````

**Comment by sarsamurmu** [comment]
Fixed in `3.17.0`
